Application/EdFi.Ods.Features/Composites/CompositeResourceModelBuilder.cs

```diff
@@ -85,7 +85,7 @@ public class CompositeResourceModelBuilder : ICompositeItemBuilder<CompositeReso
                     new ResourceProperty(
                         builderContext.CurrentResource,
                         processorContext.CurrentResourceClass.Name.ToCamelCase() + "Type",
-                        new PropertyType(DbType.AnsiString, 128, 0, 0, true),
+                        new PropertyType(DbType.AnsiString, 128, 0, 0, 0, true),
```

Comment:
As I mentioned in an earlier comment, move the `minLength` argument to the end of the constructor signature and revert all changes such as this.

Reply:
I moved to last as minLength parameter  and updated it for all places at the end .